docs/sce_file_contents.md

```diff
@@ -133,3 +133,65 @@ Finally, additional metadata for the CITE-seq data processing can be found in th
 ```r
 citeseq_metadata <- metadata(altExp(sce, "CITEseq"))
 ```
+
+## Additional `SingleCellExperiment` components for multiplexed libraries
+
+Multiplexed libraries will contain a number of additional components and fields.
+
+Hashtag oligo (HTO) quantification for each cell is included within the `SingleCellExperiment` as an "Alternative Experiment" named `"cellhash"` , which can be accessed with the following command:
+
+```r
+altExp(sce, "cellhash")
+```
+
+Within this, the main data matrix is again found in the `counts` assay, with each column corresponding to a cell or droplet (in the same order as the parent `SingleCellExperiment`) and each row corresponding to a hashtag oligo (HTO).
+Column names are again cell barcode sequences and row names the HTO ids for all assayed HTOs. 
+
+The following additional per-cell data columns for the cellhash data can be found in the main `colData` data frame (accessed with `colData(sce)` [as above](#cell-metrics)).
+
+| Column name                | Contents                                          |
+| -------------------------- | ------------------------------------------------- |
+| `altexps_cellhash_sum  `    | UMI count for cellhash HTOs                       |
+| `altexps_cellhash_detected` | Number of HTOs detected per cell (HTO count > 0 ) |
+| `altexps_cellhash_percent`  | Percent of `total` UMI count from HTO reads       |
+
+Metrics for each of the HTOs assayed can be found as a `DataFrame` stored as `rowData` within the alternative experiment:
+
+```r
+hto_info <- rowData(altExp(sce, "cellhash"))
+```
+
+This data frame contains the following columns with statistics for each HTO:
+
+| Column name | Contents                                                       |
+| ----------- | -------------------------------------------------------------- |
+| `mean`      | Mean HTO count across all cells/droplets                       |
+| `detected`  | Number of cells in which the HTO was detected (HTO count > 0 ) |
```

Comment:
Is this number of cells or is it percent of cells the HTO is detected in? I thought it was percent for gene expression so assumed it would be the same for HTO data?